### ISSUE TITLE
[Bug]: Listening parties are bugged

### ISSUE THREAD

**Issue by WildAntelope** [ciderapp/Cider-2#411]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

9/20/2023

### Operating System Version

Windows 11

### Describe the Bug

When using the party feature, where multiple users can listen along to a host's queue, a problem arises that causes the nonhost participants to not proceed in the queue after a song is either completed or skipped. For example, if the host is playing song A and skips to song B, then the rest of the party participants will hear song A being repeated from the beginning. Also, if the host is playing song C and song D is next in the queue, then once song C is finished, the host's queue moves on to play song D while song C is repeated from the beginning for the other participants. 

### Steps to Reproduce

This bug is present every time I am either the host or a participant in a listening party. 

### Build timestamp

1697238504333

**Comment by Amaru8** [comment]
Hey there! This is a duplicate of #395.
